package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "less-terrible-coffeelint-loader",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
Please don't update the version number in your PR. That will be done when a new package version is released after merging.